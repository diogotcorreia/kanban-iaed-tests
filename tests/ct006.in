t 50 mytask1
t -30 mytask1
t -30 mytask2
t 0 mytask1
t 0 mytask3
q
