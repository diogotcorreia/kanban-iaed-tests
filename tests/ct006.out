task 1
duplicate description
invalid duration
duplicate description
invalid duration
